### PR TITLE
refactoring for automation

### PR DESCRIPTION
**Pull request by jinwon0325kim**
Refactoring was performed on the code related to automation.

### CODE REVIEW

File: src/automation/automationmain.py [hunk 1]
```diff
@@ -520,74 +508,75 @@ def clear(self, start, stop):
...
+        self.logtrack.terminate_log_track_script()
+        self.logwindow("Successful commands: " +
+                        str(self.logtrack.successcount(self.currentOpenFile)))
+        if self.logtrack.filename != None:
+            self.logwindow("Test report stored at " +
+                            self.logtrack.filename)
+        self.logtrack = 0
+        self.btn_Run.setText('Run')
+        self.btn_Run.setStyleSheet(
+            'background-color: rgb(98, 160, 234);font: 16pt;font-weight:bold;')
```
Comment (spdkimo) [inline src/automation/automationmain.py:566]: I can find that line 82 is very similar with this one. Can you do refactoring for style sheet value?

**Review by spdkimo — COMMENTED** [review]
Methods are well organized with single responsibility

File: src/automation/automationmain.py (hunk 1 above)
Comment (jinwon0325kim) [reply src/automation/automationmain.py:564]: The style setting of the clear log button doesn't change, so it is set in the ui file.

**Review by spdkimo — APPROVED** [review]
LGTM

**Pull request merged**